roles/kubernetes-apps/csi_driver/vsphere/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- include_tasks: vsphere-credentials-check.yml
+  tags: vsphere-csi-driver
+
+- name: vSphere CSI Driver | Generate CSI cloud-config
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ kube_config_dir }}/{{ item }}"
+    mode: 0640
+  with_items:
+    - vsphere-csi-cloud-config
+  when: inventory_hostname == groups['kube-master'][0]
+  tags: vsphere-csi-driver
+
+- name: vSphere CSI Driver | Generate Manifests
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ kube_config_dir }}/{{ item }}"
+  with_items:
+    - vsphere-csi-controller.yml
```

Comment:
Shouldn't there be a node driver and other CSI components like here? https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/1.14/deploy/vsphere-csi-node-ds.yaml#L42
The CSI driver needs a controller, a node, and RBAC.